### PR TITLE
Run spe script on windows

### PR DESCRIPTION
**Pull request by kostobog**
Adding spe.bat and updating README.md for running the editor is windows.
Adding README-DOCKER.md summarizing installation of docker on windows focusing more on installation of docker engine and docker compose in WSL and Linux.

@blcham 
Can you revise `README.md` and make sure that `.env.custom-script-paths` file and its usage is properly explained.

### CODE REVIEW

**Review by blcham — APPROVED** [review]
@kostobog Very nicely described! Consider also approving my suggestions.

**Comment by blcham** [comment]
@rodionnv, please revise as well -- as can be seen, @kostobog created the complete guide for WSL and Powershell as well.

File: README-DOCKER.md
Comment (blcham) [inline README-DOCKER.md:1]: @kostobog I suggest moving this into ./doc/using-docker.md. And add a reference to this document from README.md.

File: README.md [hunk 1]
```diff
@@ -16,12 +16,40 @@ Default configuration assumes that:
...
+To run the editor using `docker-compose`, do the following steps:
 - `cd $PROJECT_ROOT`
-- if on Windows, create `$PROJECT_ROOT/.env` file with variable assignment `SHARED_ROOT=/host_mnt`
+- if on Windows, if not setup create `$PROJECT_ROOT/.env` according to [Windows setup .env and .env.custom-script-paths](#windows-setup-env-and-envcustom-script-paths)
 - `docker-compose up`
 - open the editor in browser at `http://localhost:3000`. The editor should be showing [example scripts from SPipes repository](https://github.com/kbss-cvut/s-pipes/doc/examples).
 
+#### Run editor using spe script
+
+To run the editor using `spe` script, do the following steps:
```
[the remark this answers is not in the document]
Comment (kostobog) [reply README.md:30]: Yes, I think is properly explained. Thanks.

File: README-DOCKER.md
Comment (kostobog) [reply README-DOCKER.md:1]: moved

**Comment by rodionnv** [comment]
> @rodionnv, please revise as well -- as can be seen, @kostobog created the complete guide for WSL and Powershell as well.

Everything works for me as described in the guide, both launching from powershell and wsl ubuntu and using spe.bat script

**Pull request merged**